src/scenarios/shared/startup.py

```diff
@@ -48,7 +48,7 @@ def __init__(self):
 
     
     def _setstartuppath(self, path: str):
-        self.startupexe = os.path.join(path, 'Startup.exe')
+        self.startuppath = os.path.join(path, 'Startup.exe' if sys.platform == 'win32' else 'Startup') 
```

Comment:
We have a helper method for this: https://github.com/dotnet/performance/blob/master/src/scenarios/shared/util.py#L29-L31

```suggestion
        self.startuppath = os.path.join(path, 'Startup%s' % extension())
```

(Or rewrite `extension` to do the string manipulation and pass in the exename.